packages/gatsby-cli/src/structured-errors/error-map.ts

```diff
@@ -596,7 +596,48 @@ const errors = {
     // TODO: change domain to gatsbyjs.com when it's released
     docsUrl: `https://v5.gatsbyjs.com/docs/reference/config-files/actions#createSlice`,
   },
-
+  "11335": {
+    text: (context): string =>
+      `${
+        context.pluginName
+      } must set the absolute path to the slice component when create creating a slice.\n\nThe (relative) path you used for the component is "${
+        context.componentPath
+      }"\n\nYou can convert a relative path to an absolute path by requiring the path module and calling path.resolve() e.g.\n\nconst path = require("path")\npath.resolve("${
+        context.componentPath
+      }")\n\nThe object passed to createSlice:\n${JSON.stringify(
+        context.component,
+        null,
+        4
+      )}\n\nSee the documentation for the "createSlice" action — https://www.gatsbyjs.com/docs/reference/config-files/actions#createSlice`,
```

Comment:
See the errors you changed above: Move the docs url to the `docsUrl` key and remove the "see the documentation" paragraph

Reply:
These were copy pasted from the `createPage` errors, I'll change those then too since this is a good opportunity